src/OpenTelemetry.Api/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 0.5.0-beta.1
+
+Released 2020-Aug-28
```

Comment:
For 0.4.0-beta we used `Released 2020-07-24`, probably need to change here to `2020-08-28`.